CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Models/OrderButtonModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace CheckerUI.Models
+{
+    public class OrderButtonModel
+    {
+        public string m_OrderButtonID { get; set; }
```

Comment:
When using a Property like this, the naming convention is without m_ (if I am not mistaken), and so it should be:
`m_OrderButtonID -> OrderButtonID`

Reply:
the private property is with m , the public Property is without m , that way  asperating  when accessing from outside